code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -218,6 +218,17 @@
 
 	if(ishuman(eater))
 		var/mob/living/carbon/human/H = eater
+		if((H.species.chem_flags & NO_EAT) && (istype(H.species,/datum/species/plasmaman)))
+			if(ismob(loc))
+				var/mob/M = loc
+				M.drop_from_inventory(src)
+			src.forceMove(get_turf(H))
+		if(istype(H.head,/obj/item/clothing/head/helmet))
```

Comment:
Didn't you mean to indent this? This code seems broken.

Also, use `get_item_by_slot(slot_head)` instead of `H.head`

Reply:
The indention was used like that for the prior skellington NO_EAT check, It didn't cause any compiling errors/warnings so I felt it safe to leave as is.
Also tested the function locally and it works as intended.